django/river/api/views.py

```diff
@@ -72,10 +72,14 @@ def create(self, request, *args, **kwargs):
 
 
 class ScriptsEndpoint(generics.ListAPIView):
+    serializer_class = serializers.ScriptsSerializer
+
     def list(self, request, *args, **kwargs):
         res = []
         for module_name, module in getmembers(scripts, ismodule):
             for script_name, script in getmembers(module, isfunction):
                 doc = getdoc(script)
-                res.append({"name": script_name, "description": doc, "category": module_name})
+                res.append(
+                    {"name": script_name, "description": doc if doc is not None else "", "category": module_name}
```

Comment:
I think `doc if doc is not None else ""` can be written `doc or ""` but I don't know if you like it or not 🤷‍♂️